src/components/RegisterUsers.tsx

```diff
@@ -53,7 +53,7 @@ export default function RegisterUsers() {
       // if the logged in user is not an Owner they are redirected to the dashboard
       if (data.userGroup === "Power" || data.userGroup === "User") {
         alert("innappropriate permissions. Please log in as administrator.");
-        history.push("/dashboard");
+        history.push("/app/");
```

Comment:
Same as above, if we need to redirect, we can just return the redirect element immediately.